data/plugins/TempHumidPlugin.html

```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
" />" for xhtml conformity. I know this is not your fault, but I fixed it in all other html files in the meantime, so please update this as well.